horace_core/symop/@SymopIdentity/SymopIdentity.m

```diff
@@ -0,0 +1,70 @@
+classdef SymopIdentity < Symop
+
+    methods
+        function obj = SymopIdentity(id, offset)
+            if ~exist('id', 'var')
+                id = eye(3);
+            end
+            if ~exist('offset', 'var')
+                offset = [0; 0; 0];
+            end
```

Comment:
not sure I understand this concept. if these args are set non-default, is it still identity?

Reply:
This allows you to define as `SymopIdentity` as:
```
SymopIdentity()
SymopIdentity(eye(3))
SymopIdentity(eye(3), [0; 0; 0])
```
Which may be useful programmatically and is provided for use with `Symop.create` and other `Symop` derivatives.